src/Microsoft.AspNetCore.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -65,7 +64,7 @@ public DateHeaderValues GetDateHeaderValues()
 
             return new DateHeaderValues()
             {
-                Bytes = _activeDateBytes ? _dateBytes0 : _dateBytes1,
+                Bytes = _dateBytes,
```

Comment:
It could be important for digest header signing I suppose? Though then you are also at the mercy of the output order with some of the headers being in a dictionary can coming out in an unknown order.